### DISCUSSION TITLE
Add sort() and scrollTo() methods for ui.table?

### DISCUSSION THREAD

**Discussion by hu76589** [zauberzeug/nicegui#1784] [Ideas / Feature Requests]
when i add row to the table,i want to sort it and scroll to the new row,now i can do it like this
```py
table.run_method('sort', 'name')
table.run_method('scrollTo', 15)
```
Perhaps there could be a more direct way to do it
```py
table.sort('name')
table.scrollTo(15)
```

Another thing, run_method() returns None,if the method we called has return value,how to receive it?
thanks!

**Comment by falkoschindler** [comment]
This sounds reasonable. Would you like to create a pull request? 🙂

Regarding the return value of `run_method`: We're planning to support return values in upcoming version 1.4 (see https://github.com/zauberzeug/nicegui/issues/1709).
